components/AddRoomModal/AddRoomModal.js

```diff
@@ -0,0 +1,126 @@
+import { useState } from "react";
+import styled from "styled-components";
+import { RoomType } from "../RoomTile/RoomTile";
+
+// Styled Components
+const OpenBtn = styled.button`
+  position: absolute;
+  top: 20px;
+  right: 50px;
+  padding: 5px 7px;
+  border: none;
+  background-color: lightgrey;
+`;
+
+const Modal = styled.div`
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  position: fixed;
+`;
+
+const Overlay = styled.div`
+  background: rgba(49, 49, 49, 0.5);
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  position: fixed;
+`;
+const ModalContent = styled.div`
+  position: absolute;
+  top: 40%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  line-height: 1.4;
+  background: #f1f1f1;
+  padding: 14px 28px;
+  border-radius: 3px;
+  max-width: 600px;
+  min-width: 300px;
+`;
+const CloseModalBtn = styled.button`
```

Comment:
Good idea, but i think i'll leave them separate for now, since the "Add Room" button on the Overview page gets a different styling later on. But if i decide to have them be the same, i'll definitely put them together in one component :)